---
description: 詳細については、' Finally ' は ' Try ' ステートメントで1回のみ使用できます
title: "'Finally' は、'Try' ステートメント内で 1 回のみ記述できます"
ms.date: 07/20/2015
f1_keywords:
- vbc30381
- bc30381
helpviewer_keywords:
- BC30381
ms.assetid: 4fa1d5fa-c54a-4f8c-9d66-9dbcc38c53bf
ms.openlocfilehash: 3b7c2873803f9af6c3d632ae331c5e26837180c1
ms.sourcegitcommit: 10e719780594efc781b15295e499c66f316068b8
ms.translationtype: MT
ms.contentlocale: ja-JP
ms.lasthandoff: 02/14/2021
ms.locfileid: "100472080"
---
# <a name="finally-can-only-appear-once-in-a-try-statement"></a>'Finally' は、'Try' ステートメント内で 1 回のみ記述できます

`Finally` は `Try...Catch...Finally` ブロックを終了するために使用します。そのため、ブロックの最後で 1 回のみ出現できます。  
  
 **エラー ID:** BC30381  
  
## <a name="to-correct-this-error"></a>このエラーを解決するには  
  
1. 不要な `Finally`を見つけて削除します。  
  
## <a name="see-also"></a>関連項目

- [Try...Catch...Finally ステートメント](../language-reference/statements/try-catch-finally-statement.md)
